mcu/nxp/lpc11u35_501.yaml

```diff
@@ -0,0 +1,22 @@
+mcu:
+    vendor:
+        - nxp
+    name:
+        - lpc11u35/501
```

Comment:
Shall we use - as separator here. Although I see uvision has /, but that's tool specific. lpc11u35-501 ?